<ui version="4.0" >
 <class>FileOptionsWidget</class>
 <widget class="QWidget" name="FileOptionsWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>112</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>File Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Number of entries in Recent Files list:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="KIntSpinBox" name="m_recentFilesEntries" >
     <property name="whatsThis" >
      <string>Controls the maximum number of filenames that are shown when you select File-> Open Recent.</string>
     </property>
     <property name="maximum" >
      <number>20</number>
     </property>
     <property name="value" >
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Autosave delay (minutes):</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="KIntSpinBox" name="m_autoSaveDelay" >
     <property name="whatsThis" >
      <string>Here you can select the time between autosaves, or disable this feature altogether by choosing Do not save automatically (drag the slider to the far left).</string>
     </property>
     <property name="specialValueText" >
      <string>Do not save automatically</string>
     </property>
     <property name="suffix" >
      <string> min</string>
     </property>
     <property name="maximum" >
      <number>60</number>
     </property>
     <property name="value" >
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QCheckBox" name="m_createBackupFile" >
     <property name="whatsThis" >
      <string>Check this box if you want some backup files created. This is checked per default.</string>
     </property>
     <property name="text" >
      <string>Create backup files</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>278</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
